addons/payment/static/src/js/payment_processing.js

```diff
@@ -117,6 +115,9 @@ odoo.define('payment.processing', function (require) {
         },
         displayContent: function (xmlid, render_values) {
             var html = Qweb.render(xmlid, render_values);
+            if ($.blockUI) {
```

Comment:
is this test needed, seeing that `displayLoading` calls `$.blockUI` unconditionally anyway?
I know it was already there, just wondering... ;-)